### ISSUE TITLE
glowroot source codes

### ISSUE THREAD

**Issue by AbleSteven** [glowroot/glowroot#553]
Hi, is there anyone could tell me why the Class cannot be resolved:
`org.glowroot.wire.api.model.DownstreamServiceOuterClass.MBeanMeta`
And, in the wire-api module there aren't any errors in the pom file and the src package
And, after I executed `mvn clean install` against on the `wire-api` module , that Class still cannot be resolved.
Looking forward to the answer !!
Thanks !!

**Comment by trask** [comment]
Hi @AbleSteven, are you getting this error in your IDE? Can you check/post your IDE error log, e.g. https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files?

**Comment by AbleSteven** [comment]
Hi @trask, forget the Class mentioned above, please. There is another Class cannot be found, that is `org.glowroot.central.repo.proto.Stored;`. And I have looked up the `org.glowroot.central.repo.proto` from all over the `central` module, and I get nothing. I followed the guide from the above url you gived to me . And I check the log file. There aren't error logs, but build logs. Because the error occurs during the building time when I run the `Main` Class of the `central` module. 
The whole reported building error by the IntelliJ Idea is bellow:
`/usr/local/idea/ideaProject/glowroot-master/central/src/main/java/org/glowroot/central/repo/SyntheticResultDaoImpl.java
Error:(40, 39) java: package org.glowroot.central.repo.proto does not exist
Error:(404, 61) java: package Stored does not exist
Error:(419, 31) java: package Stored does not exist`
Looking forward to your wisdom !!

**Comment by trask** [comment]
Hi @AbleSteven, when you created the project in Intellij, did you import it as a Maven project?

![image](https://user-images.githubusercontent.com/218610/54541503-4d566200-4957-11e9-91fe-8555bd913b3d.png)

**Comment by AbleSteven** [comment]
Of cause, I imported the source codes as a maven project!
And I finally resolved the problem. The method is : login the system using a non-root system account, and run the IntelliJ Idea, then everything runs OK!
![image](https://user-images.githubusercontent.com/43262036/54584681-f8981180-4a52-11e9-904f-55937cc44946.png)
![image](https://user-images.githubusercontent.com/43262036/54584695-0483d380-4a53-11e9-8c4f-e4fd57a6957d.png)
I appreciate your kindness!!
Thanks!!
